### PR TITLE
added user profile reducers in redux,  added testing for user profile reducer and action

### PR DESCRIPTION
**Pull request by JackyLiu97**

### CODE REVIEW

File: client/src/js/reducers/userReducer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { GET_USER_PROFILE, GET_ERRORS } from '../actions/types';
+
+const initState = {};
+
+const userReducer = (state = initState, action) => {
+    switch (action.type) {
+      case GET_USER_PROFILE:
+        return action.payload;
+      case GET_ERRORS:
+          return action.payload;
```
Comment (yliu0715) [inline client/src/js/reducers/userReducer.js]: Minor change but you can reduce those two case statements to one line:
`case GET_USER_PROFILE: case GET_ERRORS:
        return action.payload;`

File: client/src/js/actions/types.js [hunk 2]
```diff
@@ -3,3 +3,4 @@ export const FETCH_PRODUCT = "fetch_product";
 export const GET_ERRORS = "GET_ERRORS";
 export const USER_LOADING = "USER_LOADING";
 export const SET_CURRENT_USER = "SET_CURRENT_USER";
+export const GET_USER_PROFILE = "USER_PROFILE";
```
Comment (yliu0715) [inline client/src/js/actions/types.js]: should probably assign it `"GET_USER_PROFILE"` for consistency.

File: client/src/js/reducers/userReducer.js (hunk 1 above)
Comment (JackyLiu97) [reply client/src/js/reducers/userReducer.js]: ah I see. I think i will keep it as it is for completeness <3

File: client/src/js/actions/types.js (hunk 2 above)
Comment (JackyLiu97) [reply client/src/js/actions/types.js]: Good idea! consistency is good <3

**Comment by oscarchong** [comment]
Please assign reviewers to your PR or else we might over look it leading to no reviews on it because we might think it's not ready for review or something. Thank you.

**Comment by JackyLiu97** [comment]
@synzen  kek.

**Pull request merged**